features/popover.yml

```diff
@@ -2,12 +2,16 @@ name: Popover
 description: The `popover` HTML attribute creates an overlay to display content on top of other page content. Popovers can be shown declaratively using HTML, or using the `showPopover()` method.
 spec: https://html.spec.whatwg.org/multipage/popover.html
 group: html
-# TODO: https://github.com/web-platform-dx/web-features/issues/1971
-# Status changed: https://github.com/web-platform-dx/web-features/pull/1797
-# 2024-09-18 — low → false — Safari on iOS has a bug that prevents dismissing popovers by touch.
-# References:
-# - https://github.com/mdn/browser-compat-data/issues/22927
-# - https://bugs.webkit.org/show_bug.cgi?id=267688
+# notes:
+#   - date: 2024-09-18
+#     category: baseline-regression
+#     old_baseline_value: low
+#     new_baseline_value: false
+#     message: >
+#       Safari on iOS has a bug that prevents light dismiss (tapping outside the element to close it).
+#     citations:
+#       - https://bugs.webkit.org/show_bug.cgi?id=267688
+#       - https://github.com/mdn/browser-compat-data/issues/22927
```

Comment:
When I opened this PR, I had written this note. However, the feature has since progressed. It occurred to me that we should not show irrelevant notes, so I commented it out. This is a rather new idea, so I'd like to delete this entirely, if we decide this is the right way to go.

Other options considered: some sort of "historic" flag on notes that are no longer relevant, or filtering historic notes from the published package. I figured both of those added complexity that wouldn't be very useful to anyone, least of all developers.